cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -5,46 +5,488 @@
     - name: Default example
       markup: |
         <div class="o-mega-menu">
+            <button class="o-mega-menu_trigger" aria-expanded="false">
+                <span class="o-mega-menu_trigger-label">
+                    <span class="u-visually-hidden">Menu</span>
+                </span>
+            </button>
+            <nav class="o-mega-menu_content" aria-expanded="false" aria-label="main navigation" role="navigation">
```

Comment:
Stacked everything, but with indents it's still wider. Not sure what to do here but may be worth revisiting when docs are updated throughout https://github.com/cfpb/cfgov-refresh/commit/cf2ff8222c7f966e7a39d2f9af67558503a1ebc5